core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -253,7 +251,6 @@ private void configureRSM() {
         remoteLogStorageManager.configure(rsmProps);
     }
 
-    @SuppressWarnings("removal")
     RemoteLogMetadataManager createRemoteLogMetadataManager() {
         return java.security.AccessController.doPrivileged(new PrivilegedAction<RemoteLogMetadataManager>() {
             private final String classPath = rlmConfig.remoteLogMetadataManagerClassPath();
```

Comment:
I'm not too familiar with RemoteLogManager but it seems it could be interesting to check the result. So while it's not used at the moment, I wonder if we should still keep it. Otherwise I'll let it to experts in this area to gauge whether to delete it or not.

Reply:
I agree with you